MekHQ/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
         exclude group: 'org.python', module: 'jython'
         exclude group: 'org.mozilla', module: 'rhino'
     }
-    
+
```

Comment:
Sorry about the whitespace changes – I always forget my editor strips trailing spaces on save